### DISCUSSION TITLE
About DefaultKubernetesClient.java and LogMojoTest.java class

### DISCUSSION THREAD

**Discussion by Kshitiz-Mhto** [eclipse/jkube#1701] [Q&A] [answered]
I am new to jKube project and wanted to contribute, i am trying to understand the codebase, facing difficulties but  still trying .

[org.eclipse.jkube.maven.plugin.mojo.develop.LogMojoTest.java](https://github.com/eclipse/jkube/blob/master/kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/develop/LogMojoTest.java)

`line 29` in above file is like this:
```

import io.fabric8.kubernetes.client.DefaultKubernetesClient;

```
<br>

***
in this file;
[io/fabric8/kubernetes/client/DefaultKubernetesClient.java](https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/DefaultKubernetesClient.java) 
![depricated](https://user-images.githubusercontent.com/102871719/184501830-feac7400-3f5a-4c56-8ccf-7229ee286c96.png)

and this depricated Class is imported in the `LogMojoTest.java` never used inside the Test class. 

Does the `DefaultKubernetesClient` Class has any role in `LogMojoTest.java` or we can remove it?

**Comment by rohanKanojia** [comment, marked as answer]
Hi, you're right it doesn't seem to be getting used. It's safe to remove it (along with other unused imports).

Could you please raise a PR to remove these unused imports?
